[Jules]: fix remaining typescript errors

### Task Requirements

You are an Expert React and TypeScript Developer specializing in Home Assistant custom frontend panels. Your task is to resolve the final `TS2307: Cannot find module` errors preventing our Vite build from completing.

<detected_errors>
src/App.tsx(4,24): error TS2307: Cannot find module './components/DeviceView' or its corresponding type declarations.
src/App.tsx(7,28): error TS2307: Cannot find module './utils/api' or its corresponding type declarations.
src/components/EventLog.tsx(2,28): error TS2307: Cannot find module '../utils/api'
src/components/Settings.tsx(2,28): error TS2307: Cannot find module '../utils/api'
src/components/TimedAccess.tsx(3,28): error TS2307: Cannot find module '../utils/api'
src/meraki-panel.ts(3,28): error TS2307: Cannot find module './utils/api'
</detected_errors>

<instructions>
1. Check the requirements to prevent regressions and update requirements based on implemented features.
2. Think step-by-step. Recognize that Linux CI runners are strictly case-sensitive, which often causes local builds to pass but remote builds to fail.
3. Outline your strategy inside <thinking> tags. First, list the files in `custom_components/meraki_ha/www/src/components/` and `custom_components/meraki_ha/www/src/utils/` to determine the exact, case-sensitive names of the `DeviceView` and `api` files.
4. Verify the exact filenames using terminal commands before modifying imports. 
5. Update the import statements in `App.tsx`, `EventLog.tsx`, `Settings.tsx`, `TimedAccess.tsx`, and `meraki-panel.ts` to perfectly match the file casing found on the disk (e.g., if the file is `deviceView.tsx`, the import must be `./components/deviceView`).
6. Ensure that the files actually exist in the repository; ask the user to commit them if they are completely missing.
7. Output the fully refactored code for each modified file inside its own <output_code file="path/to/file.tsx"> tag.
</instructions>

<examples>
Example 1: Correcting case-sensitive imports
Input:
// File on disk: src/components/myButton.tsx
import MyButton from './components/MyButton'; // Fails on Linux

Output:
import MyButton from './components/myButton'; // Correct casing
</examples>

Please inspect the workspace directory structure for `src/components` and `src/utils` to begin.


### Specific Constraints

Ignore all errors not related to the meraki_ha integration. Follow Home Assistant Sentence Case standards.

--- a/custom_components/meraki_ha/www/dev_server.log
+++ b/custom_components/meraki_ha/www/dev_server.log
@@ -6,7 +6,7 @@
 (!) build.outDir must not be the same directory of root or a parent directory of root as this could cause Vite to overwriting source files with build outputs.
 
 
-  VITE v5.4.21  ready in 253 ms
+  VITE v5.4.21  ready in 261 ms
 
   ➜  Local:   http://localhost:5173/
   ➜  Network: http://192.168.0.2:5173/

--- a/custom_components/meraki_ha/www/index.html
+++ b/custom_components/meraki_ha/www/index.html
@@ -19,7 +19,7 @@
         document.documentElement.classList.remove('dark');
       }
     </script>
-    <script type="module" src="./src/main.tsx"></script>
+    <script type="module" crossorigin src="/meraki-panel.js"></script>
   </head>
   <body class="bg-light-background dark:bg-dark-background">
     <meraki-panel></meraki-panel>

